pkgs/applications/accessibility/contrast/default.nix

```diff
@@ -47,29 +46,32 @@ stdenv.mkDerivation rec {
     rustPlatform.rust.cargo
     rustPlatform.cargoSetupHook
     rustPlatform.rust.rustc
-    wrapGAppsHook
+    wrapGAppsHook4
     glib # for glib-compile-resources
   ];
 
   buildInputs = [
     cairo
     dbus
```

Comment:
Do we still need dbus when they switched to zbus: https://gitlab.gnome.org/World/design/contrast/-/commit/33b28cc651ecea7f61be1a081107379c5511369a